Makefile

```diff
@@ -173,6 +175,11 @@ e2e-helm-install:
 	./.staging/helm/linux-amd64/helm version --client
 
 e2e-helm-deploy: e2e-helm-install
+ifeq ($(DAPR_ANNOTATION),true)
+	@echo 'podPubsubAnnotations: {dapr.io/enabled: "true", dapr.io/app-id: "audit", dapr.io/metrics-port: "9999"}' > annotations.yaml
+else
+	@touch annotations.yaml
```

Comment:
curious, why are we creating this file if dapr annotations are not enabled?

Reply:
updated the make rule to not create empty annotation file